concepts/docs/about.md

```diff
@@ -92,6 +93,10 @@ Many modern IDEs that support Elixir can parse and display documentation and typ
 
 If a module or a function is intended for internal usage only, you can mark it with `@moduledoc false` or `@doc false`. Those modules and functions will not be included in the generated documentation. Note that that doesn't make them private. They can still be invoked and/or imported. Check the [official documentation about hiding internal modules and functions][hiding-internal-modules-and-functions] to learn about potential solutions to this problem.
 
+## Doc tests
```

Comment:
This file shows up to students only after completing the learning exercise. We need to explain doc tests in those two files as well:
- `concepts/docs/introduction.md`
- `exercises/concept/city-office/.docs/introduction.md`

Reply:
Oh, I'd been trying to figure out what was the difference between the various markdown files.